projects/cdk/utils/miscellaneous/is-array.ts

```diff
@@ -0,0 +1,4 @@
+export function tuiIsArray(value: any): value is readonly unknown[] {
+    // See: https://github.com/microsoft/TypeScript/issues/17002
+    return Array.isArray(value);
```

Comment:
I'm not sure this util makes sense. Pretty much like making `tuiRound` with just `Math.round` inside :)

Reply:
fixed